crates/iota/src/name_commands.rs

```diff
@@ -58,6 +58,8 @@ pub enum NameCommand {
     /// Auction related operations, like bidding and claiming
     #[command(subcommand)]
     Auction(AuctionCommand),
+    /// Check the availability of a domain and return its price if available
```

Comment:
```suggestion
    /// Check the availability of a domain and return its price if available.
    /// Subdomains are always free to register by the parent domain owner.
```

Maybe a little note like this would be good?

Reply:
Sure